robot/Cumulus/tests/browser/manage_households/add_address_managehh.robot

```diff
@@ -42,7 +42,11 @@ Add New Address to Household
     ...                                  object_id=&{contact1}[AccountId]
 
     Click Actions Link                   Manage Household
-    Current Page Should Be               Custom                             ManageHousehold
+    Wait Until Keyword Succeeds          1 minute
+        ...                              5 seconds
+        ...                              Current Page Should Be               Custom                             ManageHousehold
+
+
```

Comment:
ur current page keyword is already waiting for url and frame to load with timeout as 60sec, why do this again?

Reply:
modified